diff --git a/CMakeLists.txt b/CMakeLists.txt
index fbd8ac2..7e185b6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -9,7 +9,29 @@ set(KF6_MIN_VERSION "6.0.0")
 set(KDE_COMPILERSETTINGS_LEVEL "5.85")
 
 find_package(ECM ${KF6_MIN_VERSION} REQUIRED NO_MODULE)
-set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${ECM_MODULE_PATH})
+
+include(ECMFindQmlModule)
+include(ECMQmlModule)
+
+find_package(Qt6 ${QT_MIN_VERSION} REQUIRED COMPONENTS
+    Core
+    Gui
+    Quick
+    Multimedia
+    Core5Compat
+)
+find_package(KF6 ${KF6_MIN_VERSION} REQUIRED COMPONENTS
+    CoreAddons
+    Declarative
+)
+
+find_package(KF6Kirigami ${KF6_MIN_VERSION} CONFIG)
+set_package_properties(KF6Kirigami PROPERTIES
+    DESCRIPTION "A QtQuick based components set"
+    PURPOSE "Required"
+    TYPE RUNTIME
+)
 
 # Get id and version from metadata
 file(READ ${CMAKE_CURRENT_SOURCE_DIR}/package/metadata.json METADATA)
